spec/models/follow_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+require 'rails_helper'
+
+describe Follow do 
+  it "is invalid without a target_id" do
+    follow = Follow.new
+    follow.should_not be_valid
+  end
+
+  it "is invalid without a user_id" do
+    follow = Follow.new
+    follow.should_not be_valid
+  end
+end
```

Comment:
Right. the follow = Follow.new is creating the new instance, and that instance can not be valid unless it has a target_id and user_id...